### PR TITLE
Add option CleanProtectedZones to config.lua.dist

### PR DESCRIPTION
**Pull request by MillhioreBT**
Add the functionality of activate or deactivate, if you want the map to be completely cleaned or to protect the articles in protected zones.

this can be very good for custom servers that use this engine as a base.

### CODE REVIEW

File: config.lua.dist [hunk 1]
```diff
@@ -83,6 +83,7 @@ showOnlineStatusInCharlist = false
 yellMinimumLevel = 2
 yellAlwaysAllowPremium = false
 forceMonsterTypesOnLoad = true
+cleanProtectedZones = false
```
Comment (DSpeichert) [inline config.lua.dist:86]: it's called "protection zone", so this should probably be worded `cleanProtectionZones`.

**Comment by Lyuzera** [comment]
Nice idea, but this clean can be improved.
So, why #2647 is not merged? I tested it with 200+ players online and worked very well.

File: config.lua.dist (hunk 1 above)
Comment (DSpeichert) [reply config.lua.dist:86]: This is still not updated.

**Comment by nekiro** [comment]
We have that config added in #2647 so this pull request is redundant.

**Pull request closed without merging**